src/service.cpp

```diff
@@ -116,7 +116,15 @@ Service::Service(Config &config, bool test) :
             if (config.ssl.cert == "") {
                 ssl_context.set_default_verify_paths();
 #ifdef _WIN32
+#ifdef _MSC_VER
```

Comment:
I suggest using `__MINGW32__`. Also, you don't have to nest `ifdef`s.

Reply:
AppVeyor still reports failing: `C:\projects\trojan\src\service.cpp(120): error C3861: '_T': identifier not found [C:\projects\trojan\trojan.vcxproj]`.

Reply:
I figured the problem._T is provided in other header file and I didnt included it

Reply:
Wait, you deleted the env check code? Does it work in MinGW?